tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/LazyTuple.java

```diff
@@ -134,7 +134,7 @@ else if (textBytes.length <= fieldId) {
       }
       textBytes[fieldId] = null;
     } else {
-      //non-projection
+      values[fieldId] = NullDatum.get(); //non-projection
```

Comment:
This code fills the NullDatum to values array of LazyTuple when user does not insert any data on this index. However, VTuple class does not initialize the values array on itself.